src/main/kotlin/io/github/mojira/arisa/modules/PiracyModule.kt

```diff
@@ -27,7 +27,7 @@ class PiracyModule(
     }
 
     private fun assertContainsSignatures(piracySignatures: List<String>, matcher: String) = when {
-        piracySignatures.any { matcher.split("\\s+".toRegex()).contains(it) } -> Unit.right()
+        piracySignatures.any { """(?:\s|^)${Regex.escape(it)}(?:\s|$)""".toRegex().containsMatchIn(matcher) } -> Unit.right()
```

Comment:
`\b` for word boundary probably would have been better instead (though I think it's equivalent).

Reply:
Ah nice, changed now.